### ISSUE TITLE
pf-jump-links shadow root nav container axeDevTools error ('Landmarks should have a unique role or role/label/title (i.e. accessible name) combination')

### ISSUE THREAD

**Issue by rmscampos** [patternfly/patternfly-elements#2860]
## Description of the issue

I am encountering an axeDevTools issue with the pf-jump-links shadow root nav container. It doesn't have a unique aria-label and the nav semantic element is conflicting with our pfe-navigation__menu-wrapper in our primary nav element. 

### Impacted component(s)

https://patternflyelements.org/components/jump-links/

### Steps to reproduce

1. Go to https://catalog.redhat.com/cloud/detail/256907
2. Run an axeDevTools scan
3. See error 'Landmarks should have a unique role or role/label/title (i.e. accessible name) combination'

### Expected behavior

Expected to no get an error or be able to control the labels of the nav component, maybe?

### Screenshots

![Image](https://github.com/user-attachments/assets/abe20152-c59c-4781-a3ad-2ba0bd420350)

![Image](https://github.com/user-attachments/assets/61ccc691-5539-4ce9-bc3b-8d60bd896f14)

![Image](https://github.com/user-attachments/assets/3c7f17d2-888b-4621-8c45-97ca5d0ef225)

## Related issues

- #100
